specificatie/genereervariant/openapi.json

```diff
@@ -4307,6 +4330,13 @@
               "$ref" : "#/components/schemas/ZakelijkGerechtigdeHal"
             }
           },
+          "andereErfpachters" : {
```

Comment:
Wat doet dit in deze pull request?

Reply:
Ik had voor de vorige pull request geen branch aangemaakt. Beginnersfout.  Aangezien die PR nog niet gemerged is kreeg ik alle commits van die PR ook in deze branch mee. Inmiddels deze PR geclosed en worden deze wijzigingen pas in een PR gezet als #836 gemerged is. Ook checken hoe deze wijziging zich verhoudt tot #804.